src/main/resources/default/templates/biz/tycho/kb/welcome_de.html.pasta

```diff
@@ -7,13 +7,14 @@ Willkommen in der Knowledge Base von @sirius.kernel.info.Product.getProduct().ge
 </p>
 <p>
     Hier finden Sie eine Übersicht über alle Hilfeseiten und Anleitungen die wir für Sie bereitstellen. Bitte beachten
-    Sie, dass Sie auch die <a href="/open-search"><i class="fa fa-search"></i> Systemweite Suche</a> verwenden können,
-    um entsprechende Hilfeseiten zu finden.
+    Sie, dass Sie auch die <a href="/open-search" class="text-decoration-none"><i class="fa-solid fa-search"></i>
```

Comment:
Actually there is a `k:link` tag for links in KBAs. Should we use this here too?